TeamCode/src/main/java/org/firstinspires/ftc/teamcode/vision/WebcamVision.java

```diff
@@ -0,0 +1,99 @@
+package org.firstinspires.ftc.teamcode.vision;
+
+import com.acmerobotics.dashboard.FtcDashboard;
+import com.qualcomm.robotcore.hardware.HardwareMap;
+
+import org.firstinspires.ftc.robotcore.external.ClassFactory;
+import org.firstinspires.ftc.robotcore.external.Telemetry;
+import org.firstinspires.ftc.robotcore.external.hardware.camera.WebcamName;
+import org.firstinspires.ftc.robotcore.external.navigation.VuforiaLocalizer;
+import org.firstinspires.ftc.robotcore.external.tfod.Recognition;
+import org.firstinspires.ftc.robotcore.external.tfod.TFObjectDetector;
+import org.firstinspires.ftc.teamcode.R;
+
+import java.util.List;
+
+public class WebcamVision {
+
+
+    Telemetry console;
+
+    public static final String VUFORIA_LICENSE_KEY = "AcTB3h7/////AAABma7je7SvYkkqrJT5rzDrtvh/dZ4kzPKDHWZs" +
```

Comment:
Nit: nu cred ca e folosit in afara clasei asteia, asa ca propun sa fie `private`

Reply:
ok